include/zephyr/net/wifi_mgmt.h

```diff
@@ -180,6 +180,8 @@ struct wifi_iface_status {
 	enum wifi_security_type security;
 	enum wifi_mfp_options mfp;
 	int rssi;
+	unsigned short dtim_period;
+	unsigned char beacon_interval;
```

Comment:
This looks incorrect, both are swapped, but should be fixed in upstream first.

dtim_period=1-255, so, `unsigned char` 802.11-202: 9.4.2.5.1 General
beacon_interval=2 bytes, so, `unsigned short`: 802.11-2020 - 9.4.1.3 Beacon Interval field